### ISSUE TITLE
build-in virtual dyno

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#1987]
While we have https://github.com/rusefi/rusefi/issues/272 shall we build in some code to get a virtual dyno right in rusEFI?

we have VSS, we have RPM. do we need to calculate some acceleration?

https://github.com/BastianGschrey/PowerTune/issues/36 has some cool math

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/100265360-2323b880-2f1e-11eb-8fb1-e5175e41cf09.png)

**Comment by rusefillc** [comment]
progress was made under #1999 

@shadowm60 @mck1117 what is the current status of this ticket? is this waiting for real vehicle testing?

**Comment by shadowm60** [comment]
Exactly, wanted to check accel on vehicle. Next will be debug channel rename and finish hp calculation. Just fried my coils yesterday, need a few days to get new parts.

**Comment by rusefillc** [comment]
@ElDominio do you remember this dream?

**Comment by ElDominio** [comment]
2020 is before my times lol

**Comment by rusefillc** [comment]
https://github.com/rusefi/rusefi/commit/a426bee9b8162913122cc62aad31f2face529194#diff-b8a7719a120bfcfc5ab1ba434549a0505a7ce79ea7c705e50ff5dc3ef016c618

**Comment by rusefillc** [comment]
![Image](https://github.com/user-attachments/assets/5d0db6fe-33dc-42c0-8d0d-7c1e97957399)

**Comment by ggurov** [comment]
builtin virtual dyno like this is not very useful, at least not at the current stage when effort could be spent more productively elsewhere.

this functionality exists in MegaLogViewer already, in the form of calculated fields, they can use any gear, calculate from any ratio, using just RPM

![Image](https://github.com/user-attachments/assets/909db578-5639-444b-90f9-7d4a22f077fa)


![Image](https://github.com/user-attachments/assets/3ea28a52-9b94-453b-862d-24595f3c3c8c)

however, the data seems too jerky to be visible nicely in the output, likely to do with high resolution of the data in the logs, so it comes out rather choppy and spiky:

![Image](https://github.com/user-attachments/assets/8d193689-dacc-4080-aedd-f2c2251e403c)

perhaps a little bit of effort could be put towards attempting to make that work better via "smoothedRPM" field  of some sort that averages RPM for logging and using in MegaLogViewer

**Comment by 3eR01ive** [comment]
![Image](https://github.com/user-attachments/assets/cccaf86a-a309-4ea4-be88-f7820146e69a)
